### ISSUE TITLE
Add links to additional resources in each package’s docs

### ISSUE THREAD

**Issue by chasenlehara** [canjs/canjs#3713]
__tldr;__ On each documentation page, we should have a “See also” section that links to materials related to those docs: blog posts, guides, videos, and other related docs.

## The Problem

Most of the pages on the site have additional materials _elswhere_ on CanJS.com and other websites:

- Docs that are related or are used together with an API
- Blog posts that discuss the ideas behind an API
- Guides that show how to use an API
- Videos that cover a related topic

As of right now, there’s no good way to see all the additional resources available when working with a particular API. There might be a really helpful example of how to use something in one of the guides or in a blog post, but you’d have to find that yourself.

## Potential Solution

At the bottom of each page on the site, we could add a “See also” section that:

- Has a manually-curated list of links to blog posts, videos, etc.
- Has a dynamically-generated list of docs that link to that page

We already link to the APIs we’re using in our docs and guides, so it would be straightforward for us to show everything that links to the page you’re viewing.

Additionally, we could manually curate resources like blog posts and videos that we’ve published. This would give us an opportunity to link to _any_ helpful material, whether that’s additional docs on [MDN](https://developer.mozilla.org/), related third-party projects, etc.

## Alternatives

The potential solution above is just a rough draft of this idea. We would need to work with a designer to come up with the best user experience for highlighting the additional resources.

We could also not have any manual curation, and instead focus only on what we can have automatically appear on each page.

## Related issues

This idea has come up in a few other issues:

- [Add links to our blog posts to our docs](https://github.com/donejs/developer-relations/issues/31)
- [Add the can-connect videos to the site](https://github.com/canjs/canjs/issues/3103)
- [Highlight videos related to our tech on our websites](https://github.com/donejs/developer-relations/issues/21)
